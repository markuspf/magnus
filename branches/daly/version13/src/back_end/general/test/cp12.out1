lambda = 2
./cp12
0.102527  1.62921
0.181436  1.39135
0.198335  1.34511
0.350161  0.992851
2.05485  0.0328251
0.163905  1.441
0.0539987  1.79526
0.0207803  1.91858
0.119939  1.57345
1.80078  0.0545619
0.510451  0.720539
0.329799  1.03412
0.0654165  1.75473
0.398999  0.90046
1.23885  0.167873
1.00113  0.27006
0.0504243  1.80814
0.559619  0.653057
0.776201  0.423477
0.421095  0.861531

mean = 0.519935


lambda = 4
./cp12
0.126658  2.41008
0.102527  2.65431
0.181436  1.93585
0.198335  1.80932
0.28975  1.2552
0.234915  1.56304
0.350161  0.985752
0.163905  2.07648
0.0119657  3.81306
0.00940541  3.85231
0.0539987  3.22296
0.0207803  3.68096
0.119939  2.47574
0.329799  1.0694
0.0654165  3.07907
0.398999  0.810827
1.00113  0.0729324
0.0504243  3.26937
0.559619  0.426483
0.776201  0.179333

mean = 0.252268
