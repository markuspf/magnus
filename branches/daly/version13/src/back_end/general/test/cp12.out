n = 1  lambda_n = 0.102527  t_n = 0.102527  x_n = 0.956775  u_n = 0.775339
       w_n = 0  s_n = 0.956775  v_n = 3  g_n = 3  g'_n = 0  v'_n+1 = 0


n = 2  lambda_n = 0.283964  t_n = 0.181436  x_n = 0.413686  u_n = 0.215351
       w_n = 0.775339  s_n = 1.18902  v_n = 0  g_n = 2  g'_n = 1  v'_n+1 = 0


n = 3  lambda_n = 0.482299  t_n = 0.198335  x_n = 0.0772694  u_n = -0.272892
       w_n = 0.990689  s_n = 1.06796  v_n = 0  g_n = 1  g'_n = 2  v'_n+1 = 0


n = 4  lambda_n = 0.83246  t_n = 0.350161  x_n = 0.0806223  u_n = -1.97423
       w_n = 0.717798  s_n = 0.79842  v_n = 0  g_n = 0  g'_n = 3  v'_n+1 = 0


n = 5  lambda_n = 2.88732  t_n = 2.05485  x_n = 0.0979233  u_n = -0.0659813
       w_n = 0  s_n = 0.0979233  v_n = 0  g_n = 0  g'_n = 0  v'_n+1 = 4


n = 6  lambda_n = 3.05122  t_n = 0.163905  x_n = 0.113133  u_n = 0.0591346
       w_n = 0  s_n = 0.113133  v_n = 2  g_n = 2  g'_n = 0  v'_n+1 = 1


n = 7  lambda_n = 3.10522  t_n = 0.0539987  x_n = 0.0103738  u_n = -0.0104065
       w_n = 0.0591346  s_n = 0.0695084  v_n = 0  g_n = 1  g'_n = 1  v'_n+1 = 0


n = 8  lambda_n = 3.126  t_n = 0.0207803  x_n = 0.23531  u_n = 0.115371
       w_n = 0.0487281  s_n = 0.284038  v_n = 1  g_n = 1  g'_n = 2  v'_n+1 = 0


n = 9  lambda_n = 3.24594  t_n = 0.119939  x_n = 0.376625  u_n = -1.42416
       w_n = 0.1641  s_n = 0.540725  v_n = 0  g_n = 0  g'_n = 1  v'_n+1 = 2


n = 10 lambda_n = 5.04672  t_n = 1.80078  x_n = 0.341334  u_n = -0.169117
       w_n = 0  s_n = 0.341334  v_n = 0  g_n = 0  g'_n = 0  v'_n+1 = 2


n = 11 lambda_n = 5.55717  t_n = 0.510451  x_n = 0.209376  u_n = -0.120423
       w_n = 0  s_n = 0.209376  v_n = 0  g_n = 0  g'_n = 0  v'_n+1 = 1


n = 12 lambda_n = 5.88697  t_n = 0.329799  x_n = 0.0685728  u_n = 0.00315628
       w_n = 0  s_n = 0.0685728  v_n = 1  g_n = 1  g'_n = 0  v'_n+1 = 1


n = 13 lambda_n = 5.95239  t_n = 0.0654165  x_n = 0.163476  u_n = -0.235522
       w_n = 0.00315628  s_n = 0.166632  v_n = 0  g_n = 0  g'_n = 1  v'_n+1 = 0


n = 14 lambda_n = 6.35139  t_n = 0.398999  x_n = 0.358584  u_n = -0.880264
       w_n = 0  s_n = 0.358584  v_n = 0  g_n = 0  g'_n = 0  v'_n+1 = 2


n = 15 lambda_n = 7.59024  t_n = 1.23885  x_n = 0.216871  u_n = -0.784258
       w_n = 0  s_n = 0.216871  v_n = 0  g_n = 0  g'_n = 0  v'_n+1 = 1


n = 16 lambda_n = 8.59136  t_n = 1.00113  x_n = 0.152735  u_n = 0.10231
       w_n = 0  s_n = 0.152735  v_n = 1  g_n = 1  g'_n = 0  v'_n+1 = 1


n = 17 lambda_n = 8.64179  t_n = 0.0504243  x_n = 0.153434  u_n = -0.406185
       w_n = 0.10231  s_n = 0.255745  v_n = 0  g_n = 0  g'_n = 1  v'_n+1 = 0


n = 18 lambda_n = 9.20141  t_n = 0.559619  x_n = 0.121334  u_n = -0.654867
       w_n = 0  s_n = 0.121334  v_n = 0  g_n = 0  g'_n = 0  v'_n+1 = 2


n = 19 lambda_n = 9.97761  t_n = 0.776201  x_n = 0.0138344  u_n = -0.407261
       w_n = 0  s_n = 0.0138344  v_n = 0  g_n = 0  g'_n = 0  v'_n+1 = 1


n = 20 lambda_n = 10.3987  t_n = 0.421095  x_n = 0.684945  u_n = 0
       w_n = 0  s_n = 0.684945  v_n = 0  g_n = 0  g'_n = 0  v'_n+1 = 1
